test/e2e/handler/utils.go

```diff
@@ -196,8 +196,7 @@ func updateDesiredStateWithCaptureAtNodeAndWait(node string, desiredState nmstat
 	policy.WaitForAvailableTestPolicy()
 }
 
-// TODO: After we implement policy delete (it will cleanUp desiredState) we have
-//       to remove this
+// TODO: After we implement policy delete (it will cleanUp desiredState) we have to remove this.
```

Comment:
Note: gofmt does not like the earlier formatting with indentation and messed up the format of this comment, so moving it to one line to make gofmt happy